eng/packages/http-client-csharp/generator/Azure.Generator/src/AzureClientPlugin.cs

```diff
@@ -24,6 +24,10 @@ public class AzureClientPlugin : ClientModelPlugin
     /// <inheritdoc/>
     public override AzureTypeFactory TypeFactory { get; }
 
+    private AzureOutputLibrary? _azureOutputLibrary;
+    /// <inheritdoc/>
+    public override OutputLibrary OutputLibrary => _azureOutputLibrary ??= new();
```

Comment:
I think we could put `AzureOutputLibrary` here as its type
```suggestion
    public override AzureOutputLibrary OutputLibrary => _azureOutputLibrary ??= new();
```
it should be convenient if `AzureOutputLibrary` has extra properties (which is very probable).
We do the same for TypeFactory.

Reply:
updated